src/server/handlers/webhook.js

```diff
@@ -83,5 +115,23 @@ export const notify = async (req, res) => {
       logger.error(`Failed to request builds of ${repositoryUrl}: ${error}.`);
       return res.status(500).send();
     }
+  } else if (req.headers['x-launchpad-event-type'] === 'snap:build:0.1') {
```

Comment:
What does `0.1` in `snap:build:0.1` stand for? Is this some kind of webhook event type versioning?
If it changes on LP side we want to stop recognizing the event and need to reimplement?